app/src/main/java/com/termux/app/TermuxActivity.java

```diff
@@ -257,9 +272,17 @@ public void onCreate(Bundle savedInstanceState) {
         FileReceiverActivity.updateFileReceiverActivityComponentsState(this);
 
         try {
+            Intent serviceIntent;
```

Comment:
This is one issue that has already been solved locally and fix will be available in next version.